provider/data/datetime/fingerprints.csv

```diff
@@ -1,6 +1,3 @@
-*, <'static strs>, 1478B, 382 unique static strings
-datetime/buddhist/datelengths@1, <lookup>, 4104B, 171 identifiers
-datetime/buddhist/datelengths@1, <total>, 22569B, 141 unique payloads
```

Comment:
Issue: The stats are being deleted from the fingerprint file. I don't immediately see where the stats reappear. I see a comment in the .data.rs files?

Reply:
Yes, a doc comment on the macro

Reply:
will keep the totals

Reply:
Thank you!